apps/fabric-website/src/pages/Controls/ButtonPage/ButtonPage.tsx

```diff
@@ -1,12 +1,14 @@
 import * as React from 'react';
-import { Checkbox } from 'office-ui-fabric-react';
+import { Checkbox, ICheckboxStyles } from 'office-ui-fabric-react/lib/Checkbox';
```

Comment:
Having 2 exact same pages under different packages just creates risk for inconsistency. Just curious of why not reusing the `fabric-website` for our local development? Why have two sets of code to constantly update?

Reply:
At this point it's historical reasons that we haven't had time to fix. :( Totally agree with your point though.